### ISSUE TITLE
Edit box diminishes 

### ISSUE THREAD

**Issue by GuenterMuehlberger** [Transkribus/TWI-mc#62]
If I open edit status and switch between image and linebyline the edit box disappears once I come back to the image view. It re-appears only if I load the page for new.

**Comment by hackmanschorsch** [comment]
And same if you just turn the page. The edit box disapears. So please keep the edit box visible in that case. And sometimes it isn't possible to get the edit box back even with pressing View and Edit once again. So a button where you can say 'Give me the edit box' is useful.

**Comment by kansallisarkisto** [comment]
I've now made the edit box stay and also "remember" which line was previously open if you go from a segmented page to one lacking segmentation and then back (but I haven't been able to test that specific feature yet). Whilst doing this, I fixed a previously undocumented bug with edit mode becoming view mode when changing page more than once. 

Additionally, I've made the edit box placement more sensible than the browser behaviour when resizing the window. That is, its position relative to the line being edited stays the same accounting for scale. I will still open a new bug for edit dialog placement when its size relative to the window size is unmanageable.
